resources/templates/scss/report.scss

```diff
@@ -15,5 +15,8 @@ $font-size-base: .85rem !default;
 pre.text { 
     color: var(--vscode-foreground);
 }
+.list-group-item-title {
```

Comment:
Can we change scss variable values instead?

Reply:
Updated